Feat/add helm docs (#67)
caas-team/GoKubeDownscaler: website/content/docs/1 - helm-chart/2 - values/12 - podAnnotations.mdx

---
title: podAnnotations
id: podAnnotations
globalReference: docs-helm-pod-annotations
---

# podAnnotations

> The `podAnnotations` value is a map of annotations you want to add to the pods of the GoKubeDownscaler deployment.

Reviewer: ```suggestion
The `podAnnotations` value is a map of annotations that the pods of the GoKubeDownscaler deployment will have.
```
